src/Java/Lesson5/CivilAviation.java

```diff
@@ -0,0 +1,43 @@
+package Java.Lesson5;
+
+import java.util.Scanner;
+
+public class CivilAviation extends AirTransport {
+    private static int passengers;
+    int numberOfPassengers;
+    boolean businessClassAvailability;
+
+    public void trafficAir() {
+        Scanner in = new Scanner(System.in);
+        System.out.print("Введите количесиво пассажиров: ");
+        passengers = Integer.parseInt(in.nextLine());
+    }
```

Comment:
@Ihar-Malinouski лучше возвращать значение, котороt потом будем присваивать переменной экземпляра класса
```suggestion
    public static int trafficAir() {
        Scanner scan = new Scanner(System.in);
        System.out.print("Введите количесиво пассажиров: ");
        return scan.nextInt();
    }
```